themes/Monokai Vibrant-color-theme.json

```diff
@@ -539,7 +539,7 @@
     "editor.editor.findMatchBackground": "#42557B",
     "editor.findMatchHighlightBackground": "#314365",
     "editorGroup.background": "#181A1F",
-    "editorGroup.border": "#181A1F",
+    "editorGroup.border": "#333438",
```

Comment:
Similar sentiment to my other comment, I agree with this change but want to tweak the color to better match other existing color usage. Can you swap for `#3B3C40`? Good catch!

Reply:
I've tested this color, and it makes "editorGroup.border" a little bit brighter than other borders. But I suppose you're right, since it's draggable element.